kolibri_exercise_perseus_plugin/assets/src/views/index.vue

```diff
@@ -52,13 +52,16 @@
       global.$ = require('jquery');
       global.jQuery = global.$;
 
+      require('babel-polyfill');
```

Comment:
Not sure if this is the right way to do this. Didn't see any exports on the module's index, though. Seems to be working.